### PR TITLE
Add test-upgrade-submodules GA workflow

### PR DESCRIPTION
**Pull request by 2can**

### CODE REVIEW

File: .github/workflows/test-upgrade-submodules.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+jobs:
+  verify_pull_request:
+    if: github.actor == 'DevRel Bot'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get submodules
+        run: git submodule update --init --recursive
```
Comment (lkostrowski) [inline .github/workflows/test-upgrade-submodules.yml:12]: @2can I'm not sure if I understand it

What we have now:
automatic PRs that already update submodules

What we want:
Pipeline on *every* PR (including bot, but also manual) that builds docer

**Review by lkostrowski — CHANGES_REQUESTED** [review]

File: .github/workflows/test-upgrade-submodules.yml (hunk 1 above)
Comment (2can) [reply .github/workflows/test-upgrade-submodules.yml:11]: @lkostrowski OK - updated to run on every PR

Comment (lkostrowski) [reply .github/workflows/test-upgrade-submodules.yml:11]: Ensure submodules are only cloned, we can configure checkout action
https://github.com/actions/checkout

**Review by lkostrowski — APPROVED** [review]

File: .github/workflows/test-upgrade-submodules.yml (hunk 1 above)
Comment (2can) [reply .github/workflows/test-upgrade-submodules.yml]: @lkostrowski updated

**Review by lkostrowski — APPROVED** [review]

**Pull request merged**